js/app/test/json_component_interface.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@gradio/tootils";
```

Comment:
I don't think we need separate tests for Blocks and Interface since an Interface uses the same server as a Blocks

Reply:
I added them because each caused a different error actually.
* Interface: an error saying "Type is not JSON serializable: numpy.ndarray". This is fixed by https://github.com/gradio-app/gradio/pull/8041/files#diff-324a7165f5d5a8823a28b76f5653fa45f32c8144c82b2e528882c97c7eae534fR807-R812
* Blocks: in this case, there is no visually observable error but the app silently crashes because of an error at the `/info` endpoint. This is fixed by https://github.com/gradio-app/gradio/pull/8041/files#diff-324a7165f5d5a8823a28b76f5653fa45f32c8144c82b2e528882c97c7eae534fR410